### DISCUSSION TITLE
on deckgl webpage, autoselect input field on search

### DISCUSSION THREAD

**Discussion by ImDino** [visgl/deck.gl#6887] [Ideas]
As a frequent searcher on your website for the past 4 months, I find it annoying that the input field of your searchpage isn't automatically focused when entering the page.

This should be an easy fix, I would appreciate it.

**Comment by Pessimistress** [comment]
Thank you for the feedback! We are actually in the process of revamping the website. It would be greatly appreciated if you could fill in our survey here: https://forms.gle/pakGuqsZAVpPhN2C8
